tests/e2e/tests/simple/simple1_test.go

```diff
@@ -110,6 +133,33 @@ func TestSvc2Svc(t *testing.T) {
 	if len(podList) != 2 {
 		t.Fatalf("Unexpected to get %d pods when expecting 2. got %v", len(podList), podList)
 	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	g, ctx := errgroup.WithContext(ctx)
+
+	log.Infof("Configuration readiness pre-check from %v to http://echosrv.%s:8080/echo", podList, ns)
+	for i := range podList {
+		pod := podList[i]
+		g.Go(func() error {
+			for {
+				select {
+				case <-ctx.Done():
+					return ctx.Err()
+				default:
+					_, err := util.ShellContext(ctx, "kubectl exec -n %s %s -c echosrv -- /usr/local/bin/fortio load -curl http://echosrv.%s:8080/echo", ns, pod, ns) // nolint: lll
```

Comment:
you don't need the namespace
and you can use `fortio curl ...` instead of `fortio load -curl ...`